src/utils/utils.mjs

```diff
@@ -0,0 +1,52 @@
+export function filterSeancesByFeatures(seances, emptyAmount, hasEmptyVip, hasEmptyDouble) {
+  let filteredSeances = seances;
+
+  if (emptyAmount) {
```

Comment:
split it into separate methods, think of applying some kind of pattern here :)

Reply:
fixed